specification/machinelearningservices/resource-manager/Microsoft.MachineLearningServices/stable/2018-11-19/machineLearningServices.json

```diff
@@ -1222,6 +1315,105 @@
       },
       "description": "The List Virtual Machine size operation response."
     },
+    "QuotaBaseProperties": {
+      "properties": {
+        "resourceGroupName": {
+          "type": "string",
+          "title": "Name of the resource group",
+          "description": "Name of the resource group to which the resource belongs."
+        },
+        "vmfamily": {
```

Comment:
```suggestion
        "vmFamily": {
```

Reply:
Thanks for the comments, I will update those soon!